asym_crypto_yaml/crypto.py

```diff
@@ -10,6 +10,7 @@
 KEY_CHUNK_SIZE = 100
 SUPPORTED_KEY_SIZES = [2048, 4096]
 NUMBER_OF_BYTES_PER_ENCRYPTED_CHUNK = 693
+unencrypted = False
```

Comment:
I don't think you need a global variable for this.

You can just load the document without passing a private key to decrypt then loop through it and check 
     if isinstance(input, Encrypted): 

if encrypted decrypt and reencrypt and replace, otherwise skip